packages/compass-aggregations/src/components/pipeline-builder-workspace/pipeline-as-text-workspace/index.tsx

```diff
@@ -34,38 +40,45 @@ export const PipelineAsTextWorkspace: React.FunctionComponent<
 > = ({ isAutoPreview }) => {
   if (!isAutoPreview) {
     return (
+      <div className={workspaceContainerStyles}>
+        <KeylineCard
+          data-testid={containerDataTestId}
+          className={containerStyles}
+        >
+          <div className={noPreviewEditorStyles}>
+            <PipelineEditor />
+          </div>
+        </KeylineCard>
+      </div>
+    );
+  }
+  return (
+    <div className={workspaceContainerStyles}>
```

Comment:
Same here, nested the original markup inside the `div` with `workspaceContainerStyles`